prime-router/src/test/kotlin/fhirengine/engine/FhirConverterTests.kt

```diff
@@ -287,27 +298,31 @@ class FhirConverterTests {
                 topic = Topic.FULL_ELR
             )
         )
+        val testConfigPaths = HL7toFhirTranslator.Companion.configPaths.toMutableList()
+        testConfigPaths.add("./metadata/test_fhir_mapping")
+        val testTemplates = HL7toFhirTranslator.Companion.loadTemplates(testConfigPaths.toList())
 
         every { message.downloadContent() }
             .returns(validHl7)
-
-        val bundles = engine.getContentFromHL7(message, actionLogger)
-
-        mockkClass(HL7Reader::class)
         mockkObject(HL7Reader.Companion)
+        every { HL7Reader.Companion.getMessageProfile(any()) } returns testProfile
         every { HL7Reader.Companion.profileDirectoryMap[testProfile] } returns "./metadata/test_fhir_mapping"
 
-        val bundles2 = engine.getContentFromHL7(message, actionLogger)
+        mockkObject(HL7toFhirTranslator.Companion)
+        every { HL7toFhirTranslator.Companion.getMessageTemplates() } returns testTemplates
+
+        val bundles = engine.getContentFromHL7(message, actionLogger)
 
-        // the test fhir mappings produce far fewer entries than the production ones
-        assertThat(bundles2).isNotEmpty()
+        // the test fhir mappings produce a small subset of what the input HL7 contains
+        assertThat(bundles).isNotEmpty()
+        // assertThat(bundles[0].equalsDeep(FhirTranscoder.decode(expectedFHIR))).isTrue()
```

Comment:
This line was expected to be equivalent to the following assertion but does not pass. It may be worth investigating.

Reply:
let's create a ticket for this and then link the ticket in this comment before merging PR